src/MostOpenLot.java

```diff
@@ -1,19 +1,40 @@
+import java.util.Scanner;
 
 public class MostOpenLot {
 
 	public static void main(String[] args) {
 		ParkingAPI parkingAPI = new ParkingAPI();
 		
+		Scanner input = new Scanner(System.in);
```

Comment:
Hey, always make sure that you're closing this scacnner at the very end, just to keep good habits with closing streams you no longer need.

You can add the following line at the eend of the main method to do so:
```java
input.close();
```